app/Console/Commands/VariantFilesize.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\SizeVariant;
+use Illuminate\Console\Command;
+
+class VariantFilesize extends Command
+{
+	/**
+	 * The name and signature of the console command.
+	 *
+	 * @var string
+	 */
+	protected $signature = 'lychee:variant_filesize';
+
+	/**
+	 * The console command description.
+	 *
+	 * @var string
+	 */
+	protected $description = 'Set filesize of size variants if missing';
+
+	/**
+	 * Execute the console command.
+	 *
+	 * @return mixed
+	 */
+	public function handle()
+	{
+		if ($this->confirm('This command can take tens of minutes for large instances. Do you really want to run it now ?')) {
+			$variants = SizeVariant::query()
+				->where('filesize', '=', 0)
+				->get();
+
+			if (count($variants) == 0) {
```

Comment:
Here is a better approach:
```php
$variants_query = SizeVariant::query()
				->where('filesize', '=', 0);
$count = $variants_query->count();
if ($count == 0) { 
...
} 
$variants = $variants_query->get();
...
```

The difference being that if you do count on `$variants` it will fetch all the variants, hydrate the models and potentially explode the memory usage.
While if you do count() directly, the count is in SQL an no longer in PHP. Saving a large amount of time.

Also worth considering is a from / limit. That way you allow to work on smaller chunk of data.

Reply:
Thanks a lot for your suggestions, indeed it would be way better.
I changed the default value from 0 to -1 to be sure that -1 means "no value yet", because a filesize could still be 0 (even if very unlikely) and mess up with the offset/limit logic. 

Tell me if you think the command is now satisfying. :)